server/src/bots/telegram/botResponse/trendResponse.ts

```diff
@@ -1,35 +1,76 @@
 import { getCovidTrends } from '../../../services/api/api-chart';
 import { addDays, Now } from '../../../utils/dateUtils';
 import { CountrySituationInfo } from '../../../models/covid19.models';
-import { Transform } from '../../../services/domain/chart';
+import {
+    Transform,
+    enrichWithTitle,
+    enrichWithType,
+} from '../../../services/domain/chart';
 import { catchAsyncError } from '../../../utils/catchError';
 import { getRequestedCountry } from '../../../services/domain/countries';
 import { logger } from '../../../utils/logger';
 import { CallBackQueryHandlerWithCommandArgument } from '../models';
 import * as TelegramBot from 'node-telegram-bot-api';
+import { Frequency } from './../../../models/constants';
 
 export const trendsByCountryResponse: CallBackQueryHandlerWithCommandArgument = async (
     bot: TelegramBot,
     message: TelegramBot.Message,
     chatId: number,
-    requestedCountry?: string | undefined
+    requestedCountry?: string | undefined,
+    requestedFrequency?: Frequency | undefined
```

Comment:
I know that you just followed the pattern, but I recently realized that It means
requestedFrequency might be present and might not 
AND 
when it presents it could be `Frequency` and could be `undefined`. 

If that's interned, then go for that :)

Reply:
@danbilokha 
Could you please paste what I should do with it?
Are you saying that undefined is redundant?
And I should remove it to be ```requestedFrequency?: Frequency```

Reply:
just told you how it works so you were aware of that.